google/cloud/security/common/gcp_api/compute.py

```diff
@@ -47,23 +47,91 @@ def __init__(self, credentials=None, version=None):
     # TODO: Migrate helper functions from gce_firewall_enforcer.py
     # ComputeFirewallAPI class.
 
-    def get_backend_services(self, project_id):
-        """Get the backend services for a project.
+    @classmethod
```

Comment:
Hmmm, is there a reason to use ```@classmethod``` here?  I see this is being invoked via class instances ```self._flatten_aggregated_list()``` which would make ```@classmethod``` to be not needed.